spec/PeriodicBackgroundSync-index.bs

```diff
@@ -64,21 +65,22 @@ The <code>periodicsync</code> event is considered to run <dfn>in the background<
 
 The user agent is considered to be [=online=] if the user agent has established a network connection. A user agent MAY use a stricter definition of being [=online=]. Such a stricter definition MAY take into account the particular [=service worker=] or origin a [=periodicsync registration=] is associated with.
 
-#TODO: link to PeriodicSyncManager.register().
-A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent.
+A <dfn>periodic Background Sync opportunity</dfn> allows periodic synchronization between the server and the web app, the exact interval of which is decided by the user agent. This can be requested through {{PeriodicSyncManager}}.
```

Comment:
You should refer to the actual method here, not the interface on which the method is hosted.

Reply:
done.

Reply:
Do you still have to upload a new commit to this PR? It's not resolved yet. I'll hold off reviewing until that's done :).